### PR TITLE
Fix send email to contact If send email to user action do not choose user

### PR DESCRIPTION
**Pull request by julienWebmecanik**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no

#### Description:
When creating a form with "send email to user" action without choose user, then email is send to contact

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. create a form
3. add send email to user action
4. Choose email but don't choose use
5. Submit form and check you as contact receive email
6. After PR you have to choose user in Send email to user action

### CODE REVIEW

File: app/bundles/EmailBundle/Form/Type/FormSubmitActionUserEmailType.php [hunk 1]
```diff
@@ -42,7 +43,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'class'   => 'form-control',
                     'tooltip' => 'mautic.core.help.autocomplete',
                 ],
-                'required' => false,
+                'required'    => true,
+                'constraints' => new NotBlank(),
```
Comment (kuzmany) [inline app/bundles/EmailBundle/Form/Type/FormSubmitActionUserEmailType.php]: I think we should use for support translations

```
'constraints' => [
                new NotBlank(
                    [
                        'message' => 'mautic.core.value.required',
                    ]
                ),
            ],
```

**Review by fedys — DISMISSED** [review]
Good job @julienWebmecanik! Thank you.
I tested the change and it is working fine.

validation

**Review by npracht — DISMISSED** [review]
👍🏽

**Comment by RCheesley** [comment]
Looks like we need a small bump to the code coverage on this one and then we are good to go @julienWebmecanik :)

**Comment by kuzmany** [comment]
@RCheesley unit tests added. After pass, could be merged.

**Comment by RCheesley** [comment]
@all-contributors please add @julienWebmecanik for code

**Comment by RCheesley** [comment]
@all-contributors please add @fedys for userTesting

**Comment by RCheesley** [comment]
@all-contributors please add @kuzmany for tests

**Pull request merged**